src/main/java/edu/unc/lib/boxc/migration/cdm/services/SipService.java

```diff
@@ -187,7 +187,7 @@ public List<MigrationSip> generateSips(SipGenerationOptions options) {
                 }
             }
             DisplayProgressUtil.finishProgress();
-            redirectMappingService.addCollectionRow(sips);
+            redirectMappingService.addCollectionRow(options, sips);
```

Comment:
Rather than passing in the SipGenerationOptions and creating a dependency on it there, maybe we should wrap this method call in a `options.isSuppressCollectionRedirect` check instead. In that case we could revert the changes in RedirectMappingService and keep the suppression logic more isolated